boards/egis/egis_et171/egis_et171_defconfig

```diff
@@ -0,0 +1,34 @@
+CONFIG_XIP=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Serial driver options
+CONFIG_SERIAL=y
+
+# GPIO driver options
+CONFIG_GPIO=y
+
+# Syscon driver options
+CONFIG_SYSCON=y
```

Comment:
No need for this, it will be removed for `adp_xc7k_ae350` since it is not used during platform initialization.